hawkular-accounts-feature-pack-resources/src/main/resources/subsystem-templates/hawkular-accounts-keycloak-adapter.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+
+    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<!--  Template used by WildFly build when directed to include Keycloak subsystem in a configuration. -->
+<config>
+  <extension-module>org.keycloak.keycloak-adapter-subsystem</extension-module>
+  <subsystem xmlns="urn:jboss:domain:keycloak:1.1">
+    <realm name="hawkular">
+      <auth-server-url>http://localhost:8080/auth</auth-server-url>
+      <ssl-required>none</ssl-required>
+    </realm>
+    <secure-deployment name="hawkular-accounts.war">
+      <realm>hawkular</realm>
+      <resource>hawkular-accounts-backend</resource>
+      <use-resource-role-mappings>true</use-resource-role-mappings>
+      <enable-cors>true</enable-cors>
+      <enable-basic-auth>true</enable-basic-auth>
+      <credential name="secret">9fd5e75c-9e39-4091-9fc9-085e3a7ad335</credential>
```

Comment:
A new UUID is already being generated here: https://github.com/ppalaga/hawkular-accounts/blob/151202-feature-pack/pom.xml#L291-L296 . On the original standalone.xsl, there is (was?) a XSL parameter, that is set by the assembly plugin. So, the final artifact contains a new UUID for each build.